packages/core-plugin-api/src/apis/definitions/AppI18nApi.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ApiRef, createApiRef } from '../system';
+import { Observable } from '@backstage/types';
+import {
+  InitOptions,
+  Namespace,
+  Resource,
+  ResourceKey,
+  TFunction,
+  i18n,
+} from 'i18next';
+
+export type { Resource, i18n } from 'i18next';
+
+export type Locals = {
+  [lng: string]: ResourceKey;
+};
+
+export type TFC<N extends Namespace = string, TKPrefix = undefined> = TFunction<
+  N,
+  TKPrefix
+>;
+
+export type LocalConfig = InitOptions & {
+  lazyResources?: () => Promise<Resource>;
+  lngOptions?: Set<string> | string[];
+};
+
+/**
+ * The AppI18nApi gives access to the current app i18n, and allows switching
+ * to other options that have been registered as a part of the App.
+ *
+ * @public
+ */
+export type AppI18nApi = {
+  instance: i18n;
+  /**
+   * Get a instance of i18next.
+   */
+  getI18n(): i18n;
+
+  /**
+   * Get all support languages
+   */
+  getLanguages(): string[];
+
+  /**
+   * Observe the currently language.
+   */
+  activeLng$(): Observable<string>;
+
+  /**
+   * Get the current language.
+   */
+  getActiveLng(): string;
```

Comment:
These should probably say Langage instead of Lng

Reply:
ok，I will change this